packages/desmos-std/src/profiles/msg_router.rs

```diff
@@ -0,0 +1,189 @@
+use crate::{profiles::msg_routes::ProfilesMsgs, types::DesmosRoute};
+use cosmwasm_std::{Addr, CosmosMsg, CustomMsg};
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
+#[serde(rename_all = "snake_case")]
+pub struct ProfilesMsgRouter {
+    pub route: DesmosRoute,
+    pub msg_data: ProfilesMsgs,
+}
+
+impl Into<CosmosMsg<ProfilesMsgRouter>> for ProfilesMsgRouter {
+    fn into(self) -> CosmosMsg<ProfilesMsgRouter> {
+        CosmosMsg::Custom(self)
+    }
+}
+
+impl CustomMsg for ProfilesMsgRouter {}
+
+pub struct ProfilesMsgBuilder {}
+
+impl ProfilesMsgBuilder {
+    pub fn new() -> Self {
+        ProfilesMsgBuilder {}
+    }
+}
+
+impl Default for ProfilesMsgBuilder {
+    fn default() -> Self {
+        Self::new()
+    }
+}
+
+impl ProfilesMsgBuilder {
```

Comment:
Should we test the msg builder?

Reply:
Why not, it's trivial to test them and it's a good way to check unwanted changes